AWS-CodePipeline/two-environment-pipeline-template/questions.json

```diff
@@ -0,0 +1,150 @@
+{
+  "questions": [{
+    "key": "git_provider_type",
+    "question": "What is the Git provider?",
+    "options": ["Bitbucket", "CodeCommit", "GitHub", "GitHubEnterpriseServer"],
+    "nextQuestion": {
+      "CodeCommit": "codecommit_repository_name"
+    },
+    "defaultNextQuestion": "codestar_full_repository_id"
+  }, {
+    "key": "codestar_full_repository_id",
+    "question": "What is the full repository id (Example: some-user/my-repo)?",
+    "isRequired": true,
+    "defaultNextQuestion": "main_git_branch"
+  }, {
+    "key": "codecommit_repository_name",
+    "question": "What is the CodeCommit repository name?",
```

Comment:
Just to make sure, it is name not ARN?

Reply:
See https://github.com/aws/aws-sam-cli-pipeline-init-templates/pull/18#discussion_r657298391